server/src/http/feature_refresher.rs

```diff
@@ -1199,6 +1202,7 @@ mod tests {
             last_refreshed: None,
             last_check: None,
             failure_count: 0,
+            last_feature_count: 0,
```

Comment:
An Option surely? (Yes I'm calling you Sirely).

I'd rather know if there's no update but I think the adjacent critical fix is more important so I won't fuss

Reply:
Agreed, fixing.